### PR TITLE
update third-party dependency versions based on security scans

### PR DESCRIPTION
**Pull request by davidangb**
no Jira ticket.

security scans showed warnings on two third-party libraries. I've upgraded both of those.

I have not run unit tests locally - I can't get those running on develop branch, much less this one. I do see the tests pass in Travis for this PR, though.

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -24,12 +24,13 @@ object Dependencies {
...
   val jacksonCore: ModuleID =        "com.fasterxml.jackson.core" % "jackson-core"        % jacksonV
 
-  val logbackClassic: ModuleID = "ch.qos.logback"             %  "logback-classic" % "1.2.1"
+  val logbackClassic: ModuleID = "ch.qos.logback"             %  "logback-classic" % "1.2.2"
   val ravenLogback: ModuleID =   "com.getsentry.raven"        %  "raven-logback"   % "7.8.6"
   val scalaLogging: ModuleID =   "com.typesafe.scala-logging" %% "scala-logging"   % scalaLoggingV
   val swaggerUi: ModuleID =      "org.webjars"                %  "swagger-ui"      % "2.2.5"
   val ficus: ModuleID =          "com.iheart"                 %% "ficus"           % "1.4.0"
   val cats: ModuleID =           "org.typelevel"              %% "cats"            % "0.9.0"
+  val httpClient: ModuleID =     "org.apache.httpcomponents"  % "httpclient"       % "4.5.3"  // upgrading a transitive dependency to avoid security warnings
```
Comment (rtitle) [inline project/Dependencies.scala]: Should we exclude this from whatever is pulling it in transitively?

Comment (rtitle) [reply project/Dependencies.scala]: Looks like from google libraries?
```
[info]   +-org.broadinstitute.dsde.workbench:workbench-google_2.12:0.3-97e266d [S]
[info]   | +-com.google.apis:google-api-services-admin-directory:directory_v1-rev82-1.22.0
[info]   | | +-com.google.api-client:google-api-client:1.22.0
[info]   | |   +-com.google.http-client:google-http-client-jackson2:1.22.0
[info]   | |   | +-com.fasterxml.jackson.core:jackson-core:2.1.3 (evicted by: 2.9.0)
[info]   | |   | +-com.fasterxml.jackson.core:jackson-core:2.9.0
[info]   | |   | +-com.google.http-client:google-http-client:1.22.0
[info]   | |   |   +-com.google.code.findbugs:jsr305:1.3.9 (evicted by: 3.0.0)
[info]   | |   |   +-com.google.code.findbugs:jsr305:3.0.0
[info]   | |   |   +-org.apache.httpcomponents:httpclient:4.0.1
```

Comment (davidangb) [reply project/Dependencies.scala]: Do you prefer to have the exclusions in place, or is this an open question?

I generally prefer to NOT add the excludes, unless there's a conflict. I tried `sbt assembly` on this and did not see a conflict. Unless we really need the exclude, I find them to be just more code to maintain.

All that said, I have the exclusions working locally - let me know which you'd prefer!

Comment (rtitle) [reply project/Dependencies.scala]: Thanks - given that sbt does the right thing, it's fine to leave out the excludes.

**Pull request merged**